packages/block-editor/src/components/block-settings-menu/block-visually-convert-button.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { rawHandler, getBlockContent } from '@wordpress/blocks';
+import { ToolbarButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+export default function BlockVisuallyConvertButton( { clientIds, ...props } ) {
```

Comment:
`BlockVisuallyConvertButton` feels like a bit of an awkward name. There's not really any "conversion" happening. How about `BlockEditVisuallyButton`?

Reply:
@ZebulanStanphill @adamziel sure, why not. I've already changed the name